projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2893,6 +2893,15 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'jp_group'          => 'videopress',
 			),
 
+			// Writing Prompts.
+			'jetpack_blogging_prompts_enabled'     => array(
+				'description'       => esc_html__( 'Displays a writing prompt when starting a new post.', 'jetpack' ),
+				'type'              => 'boolean',
+				'default'           => jetpack_has_write_intent() || jetpack_has_posts_page(),
```

Comment:
Ideally, we'd factor this check to a helper function since we're doing it in multiple places. Something like `jetpack_get_blogging_prompts_default`? This is not blocking though.